nodejs/cloud-init/scripts/kube-vip-lb-services.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -e
+
+echo "Install kube-vip for Load Balancing..."
+
+IMAGE=ghcr.io/kube-vip/kube-vip:v0.6.0
```

Comment:
Here we're not using the variable set up in the other script. Which is fine, just weird that it's customizable in one and not the other.

Reply:
addressed in https://github.com/equinix-labs/pulumi-equinix-kubernetes-cluster/pull/9/commits/a0b9506847dc23d818987ed5bc592684addac079